packages/dashboard/src/dashboard-actions/redo.spec.ts

```diff
@@ -228,3 +231,56 @@ describe('UPDATE', () => {
     ).toEqual(30);
   });
 });
+
+describe('MULTIPLE ACTIONS', () => {
+  it('redoes a move action and create action', () => {
+    const redoState = dashboardReducer(
+      { ...state },
+      onMoveAction({
+        position: { x: 10, y: 10 },
+        prevPosition: { x: 2, y: 2 },
+        isActionComplete: true,
+      })
+    );
+    expect(
+      redo({
+        dashboardAction: onMoveAction({
+          position: { x: 10, y: 10 },
+          prevPosition: { x: 2, y: 2 },
+          isActionComplete: true,
+        }),
+        dashboardState: redo({
+          dashboardState: { ...state },
+          dashboardAction: onCreateAction({ widgets: [MOCK_LINE_CHART_WIDGET] }),
+        }),
+      }).dashboardConfiguration.widgets
+    ).toEqual(
+      dashboardReducer({ ...redoState }, onCreateAction({ widgets: [MOCK_LINE_CHART_WIDGET] })).dashboardConfiguration
+        .widgets
+    );
+  });
+  it('redoes a resize action and create action', () => {
+    let redoState = dashboardReducer(
+      { ...state },
+      onResizeAction({
+        isActionComplete: true,
+        anchor: 'bottom-right',
+        changeInPosition: { x: 20, y: 20 },
+      })
+    );
+    redoState = dashboardReducer({ ...redoState }, onCreateAction({ widgets: [MOCK_LINE_CHART_WIDGET] }));
```

Comment:
rather than altering the reference, just make a new reference - variables are cheap! avoid mutations